### DISCUSSION TITLE
Can't get videos to play

### DISCUSSION THREAD

**Discussion by abinocoe** [react-static/react-static#1595] [Q&A] [answered]
Hi, I need to put a video carousel on my react-static site, but even before I get to that step I can't get a video to play. I've tried using react-player, I've tried the HTML video tag, all to no avail. I want to play an mp4 file that I've included in the project, it's definitely bundling and can view the video at the static url that I get out of the empty rendered component when I visit it at http://localhost:3000/static/"mymp4link" but I'm out of ideas. Any help greatly appreciated!

**Comment by abinocoe** [comment, marked as answer]
~Answered my own question, no idea how to delete this discussion now 😅~

the component is rendering a div, rather than a video tag. that's the element the src has been placed on.

**Reply by SleeplessByte** [reply]
I've moved your answer here. All good!
